maro/simulator/scenarios/finance/account.py

```diff
@@ -0,0 +1,39 @@
+from collections import OrderedDict
+
+from maro.backends.frame import NodeAttribute, NodeBase, node
+from maro.simulator.scenarios.finance.common import Order, TradeResult, two_decimal_price
+
+
+@node("account")
+class Account(NodeBase):
+    """Account node definition in frame, used to maintain cash changing."""
+    remaining_cash = NodeAttribute("f")
```

Comment:
how about set ``remaining_cash`` and ``net_asset_value`` as "private", and implement corresponding read-only property for them?

Reply:
seems can't do this for they are node attribute